### PR TITLE
Add stack horizontal

### PR DESCRIPTION
**Pull request by jeswan**
This PR does not yet support color input images (assumes input images are bw).

Tested using:
`python src/models/model.py --data_dir '/scratch/js11133/capstone/daVinci' --gpus 1 --frames_per_sample 2 --batch_size 4 --lr 0.001 --limit_train_batches=0.01 --limit_val_batches=0.01 --max_epochs=1 --stack_horizontal --output_img_freq=10`
and
`python src/models/model.py --data_dir '/scratch/js11133/capstone/daVinci' --gpus 1 --frames_per_sample 1 --batch_size 4 --lr 0.001 --limit_train_batches=0.01 --limit_val_batches=0.01 --max_epochs=1 --stack_horizontal --output_img_freq=10 --include_right_view`

### CODE REVIEW

File: src/data/data.py [hunk 1]
```diff
@@ -66,12 +64,15 @@ def __getitem__(self, index):
                 image = self.img_transform(image)
                 right_images.append(image)
 
-        # channels are the b/w input frames
-        image_tensor = torch.cat(images)
+        if self.stack_horizontal:
+            stack_dim = 2
+        elif not self.stack_horizontal:
+            stack_dim = 0
```
Comment (jeswan) [inline src/data/data.py:70]: specify which dimension to stack along

File: src/models/unet.py [hunk 2]
```diff
@@ -39,7 +40,7 @@ def __init__(
         self.layers = nn.ModuleList(layers)
 
         # final layer to match the depth map dimensions
-        self.conv_reshape = nn.Conv2d(in_channels=1, out_channels=1, kernel_size=(3,11), stride=1, padding=1, dilation=8)
+        self.conv_reshape = nn.Conv2d(in_channels=1, out_channels=1, kernel_size=(3,11), stride=(1, num_stack_horizontal), padding=(1, num_stack_horizontal), dilation=(8, 8*num_stack_horizontal))
```
Comment (annikabrundyn) [inline src/models/unet.py:43]: @jeswan did you check that if you add this line the normal code still runs with and without stack horizontal? will try run the 2 experiments you tried this on as well as running same code with stacking set to False

Comment (jeswan) [reply src/models/unet.py:43]: Yes! I checked it works with the default arguments

Comment (annikabrundyn) [reply src/models/unet.py:43]: thats awesome!

**Comment by annikabrundyn** [comment]
does the logging work as is too?

**Pull request merged**